kroxylicious-api/src/main/java/io/kroxylicious/proxy/config/tls/ServerOptions.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.config.tls;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+
+/**
+ * Options that apply to the TLS peer when in server mode.
```

Comment:
personally I have to think twice every time I read this description because I find it ambiguous as to which thing is the client and server. 
 
```suggestion
 * Options that control TLS negotiation when in server mode.
```

I was initially wondering about 
```suggestion
 * Options that are applied to the TLS peer by kroxylicious.
```

but now I don't know which makes more sense